mlrun/datastore/targets.py

```diff
@@ -1113,6 +1122,112 @@ def is_single_file(self):
         return True
 
 
+class SnowflakeTarget(BaseStoreTarget):
+    """
+    Snowflake SQL Target
+
+    :param name: Target name
```

Comment:
I don't really see any value in parameter documentation that states the obvious.